Resources/config/routing.xml

```diff
@@ -34,8 +34,11 @@
         <service id="emsch.controller.router" class="EMS\ClientHelperBundle\Controller\RouterController">
             <argument type="service" id="emsch.routing.handler" />
             <argument type="service" id="twig"/>
+            <argument type="service" id="ems_common.storage.processor"/>
             <tag name="controller.service_arguments"/>
         </service>
+        <service id="EMS\ClientHelperBundle\Controller\RouterController" alias="emsch.controller.router" public="true" />
```

Comment:
why do we need this alais?

Reply:
43289684af3c16a50cbe627ff924d4bf8efa8b7a